openedx/core/djangoapps/user_api/views.py

```diff
@@ -669,26 +669,58 @@ def _add_terms_of_service_field(self, form_desc, required=True):
         terms_text = _(u"Terms of Service")
         terms_link = u"<a href=\"{url}\">{terms_text}</a>".format(
             url=marketing_link("TOS"),
-            terms_text=terms_text
+            terms_text=terms_text,
```

Comment:
> remove this diff

To clarify, @caseylitton , every line which we commit we do so explicitly. In this case, this change is:
- unnecessary
  - Sure, it's the preferred style (trailing comma), but it doesn't actually add anything substantive in this context.
- unrelated
  - There are times where it makes sense to clean up old syntax, but that goal is outside the scope of this pull request; PRs should be focussed.
- divergent
  - It's one more difference between our code and upstream. Not only does it add noise when comparing origin and upstream, but more importantly, it increases the odds of hitting Yet Another Merge Conflict.